quickcheck-plutus-instances/src/Test/QuickCheck/Plutus/Instances.hs

```diff
@@ -422,6 +454,32 @@ instance Function PubKeyHash where
       into :: PubKeyHash -> BuiltinByteString
       into (PubKeyHash bs) = bs
 
+-- | @since 2.1
+deriving via PubKeyHash instance Arbitrary PaymentPubKeyHash
+
+-- | @since 2.1
+deriving via PubKeyHash instance CoArbitrary PaymentPubKeyHash
+
+-- | @since 2.1
+instance Function PaymentPubKeyHash where
```

Comment:
Just curious: what's wrong with the default instance of `Function`?

Reply:
There isn't one? You can't `via`-derive it.

Reply:
I was talking about https://hackage.haskell.org/package/QuickCheck-2.14.2/docs/Test-QuickCheck.html#v:function

Reply:
It requires `Generic`, mainly. It could be used here, but I am _not_ a fan of it; at least this way, we can know what's happening for sure.